.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
+  "printWidth": 120,
```

Comment:
We can run `yarn pretty:fix` in a separate PR to fix all the other TS files. And add the prettier check to CI then.